site/en/workflows.md

```diff
@@ -1028,8 +1036,9 @@ elements etc. Optionally, the output can be converted to other formats, or copie
 
 ### Step 18: Adaptation of Coordinates
 Some processors change the original image by e.g. scaling, deskewing or cropping it and produce alternative images, whose coordinates differ
-from the original images. Nevertheless, all output from the OCR-D-Processors sticks to the coordinates of the original images. If you want to 
-visualize the output with the produced alternative images, the coordinates have to be re-calculated accordingly.
+from the original images. However, OCR-D processors adhere to the coordinates of the original image. To work with these alternative images,
+such as visualizing them in a GUI, the coordinates have to be translated from the original image. `ocrd-segment-repair` exchanges the original
+(`@filename`) image with one of the alternative images\` <code>@imageFilename</code> and thereby translates the coordinates.
```

Comment:
That's not what I [suggested](https://github.com/OCR-D/ocrd-website/pull/121#discussion_r451735509).

Also, it's more incorrect than before (`ocrd-segment-repair` does nothing of the kind)...

Reply:
I now combined your's and @kba 's suggestion. Hopefully it's correct and understandable now

Reply:
LGTM now.

Reply:
Sorry, but I still cannot give my blessing for that paragraph. It's simply not true that 
> Some processors change the original image

or at least a very confusing choice of words. And if all this paragraph is good for is motivating the next one (on `ocrd-segment-replace-original`), then why not take my suggestion as it is?

Reply:
Oh no, already merged...

Reply:
Fear not, easily remedied in master.

Reply:
done

Reply:
> Fear not, easily remedied in master.

It's a complicated subject, so yes, let's open a dedicated PR for how to attack this in the documentation. (Picture might also change soon when we finally get some actual page-level dewarping...)

Reply:
> done

I'm too slow today I guess

Reply:
anyway, thx @kba @EEngl52 !

Reply:
We will need to update it soon again, probably but one of the issues adressed here is to list `anybaseocr` processors last, that was pressing because we made the impression that we endorsed it because they were listed first in their resp. categories.